lib/elasticity/search.rb

```diff
@@ -335,6 +335,18 @@ def next_page
       def previous_page
         current_page > 1 ? (current_page - 1) : nil
       end
+
+      def error
+        @response["error"]
+      end
+
+      def exception
+        @response["exception"]
+      end
+
+      def valid?
+        !error.present?
```

Comment:
I think rubucop will suggest you to use `error.blank?` instead of `error.!present?`